tests/performance/perf.find.js

```diff
@@ -0,0 +1,123 @@
+'use strict';
+
+module.exports = function (PouchDB, opts, callback) {
+
+  var utils = require('./utils');
+
+  function makeTestDocs() {
+    var docs = [];
+    for (var i = 0; i < 1000; i++) {
+      docs.push({
+        key: i % 47,
+        even: i % 2 === 0,
+        str: ['foo', 'bar', 'smang', 'foobar'][i % 3]
+      });
+    }
+    return docs;
+  }
+
+  var testCases = [
+    {
+      name: 'create-index',
+      assertions: 1,
+      iterations: 1,
+      setup: function (db, callback) {
+        db.bulkDocs(makeTestDocs())
+          .then(function () {
+            callback();
+          }, callback);
+      },
+      test: function (db, itr, doc, done) {
+        db.createIndex({
+          index: {
+            fields: ['key']
+          }
+        }).then(function () {
+          done();
+        }, done);
+      }
+    },
+    {
+      name: 'simple-find-query',
```

Comment:
Could you add another test like this `simple-find-query` but it doesn't use an index but rather just `_all_docs` - basically just don't do the createIndex part. It would be nice to have that as a comparison to see how it compares.

Reply:
Done :-)